tests/flow/test_index_create.py

```
@@ -165,13 +165,23 @@ def test02_fulltext_index_creation_label_config(self):
             self.env.assertContains("Nostem must be bool", str(e))
 
         try:
-            # phonetic must be boolean
+            # phonetic accepts either a bool or the 'dm:en' algorithm
+            # code. Any other string is rejected as unsupported.
             result = self.graph.query(
                 "CREATE FULLTEXT INDEX FOR (n:L7) ON (n.p1) OPTIONS {phonetic: 'true'}"
             )
             assert False
```

Ruff (0.15.11)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)